gateway/ln-gateway/proto/gatewaylnrpc.proto

```diff
@@ -25,11 +25,15 @@ service GatewayLightning {
    * to response with `InterceptHtlcResponse` after the HTLC has been processed with
    * the appropriate action (Settle, Forward, Cancel).
    */
-  rpc RouteHtlcs(stream InterceptHtlcResponse) returns (stream InterceptHtlcRequest) {}
+  rpc RouteHtlcs(EmptyRequest) returns (stream InterceptHtlcRequest) {}
+
+  rpc CompleteHtlc(InterceptHtlcResponse) returns (EmptyResponse) {}
```

Comment:
This seems like a much better API design to me.

Reply:
We actually originally had it like this, but then thought streaming would simplify the code a bit. Learned in the process that there's some issues doing that :)